lib/kitchen/directions/bake_link_placeholders.rb

```diff
@@ -26,6 +26,14 @@ def self.v1(book:, cases: false)
 
           if replacement.present?
             anchor.replace_children(with: replacement)
+            if label_wrappers && anchor.has_class?('lo-reference')
+              anchor.wrap_children('span', class: 'label-counter')
+              anchor.prepend(child:
+                <<~HTML
+                  <span class="label-text">#{I18n.t(:lo_label_text)}</span>
+                HTML
+              )
+            end
```

Comment:
This seems to be a pretty specific case. The `label_wrappers` argument doesn't immediately make it clear that this argument is basically just to add the `LO` label to learning objectives links. 

Instead of doing this within `BakeLinkPlaceholders`, I think a separate `BakeLOLinkLabel` direction could handle this. That way, it's easier to remember what the direction is for, and keeps it from getting buried in `BakeLinkPlaceholders`.